src/main/java/org/ajoberstar/gradle/bintray/BintrayRules.java

```diff
@@ -11,35 +26,44 @@
 import org.gradle.api.publish.PublishingExtension;
 
 class BintrayRules extends RuleSource {
-    @Model
-    public void bintray(Bintray bintray) {
-        // just create the model element
-    }
+  @Model
+  public void bintray(Bintray bintray) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=squid%3AIndentationCheck)